### DISCUSSION TITLE
Export Submissions to Excel

### DISCUSSION THREAD

**Discussion by The-Real-Thisas** [docusealco/docuseal#5] [Ideas]
It would be immensely helpful to provide an export option in the dashboard.

**Comment by The-Real-Thisas** [comment]
I think it's here app/views/submissions/show.html.erb

**Reply by The-Real-Thisas** [reply]
I would add it myself but I've never really worked with ruby, I think I could add it if someone could give me like a general idea of how it works and what is needed.

**Reply by AlexBTurchyn** [reply]
Thanks, I've been thinking about this feature as well. Hopefully, it will be added in the near future. I'll keep you updated.

**Reply by The-Real-Thisas** [reply]
Thanks !!! Lmk any way I could help because I think this is a really neat project.

**Comment by AlexBTurchyn** [comment]
@The-Real-Thisas Can you please elaborate - is that to export one particular submission in json? or to export all submissions in csv format?

**Reply by The-Real-Thisas** [reply]
Ideally something like excel because like there needs to be images in the cells right ? But csv would be the easiest to add and have the images as base64.

**Reply by The-Real-Thisas** [reply]
@AlexandrToorchyn  Ohh yea all the submissions like in bulk. Oh and sorry misread.

**Comment by AlexBTurchyn** [comment]
@The-Real-Thisas In the latest release (1.1.5), we added the ability to export submissions to CSV/XLSX formats. Looking forward to your feedback
